PredixMobileUIKit/PredixBarChartView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  PredixBarChartView.swift
+//  PredixMobileUIKit
+//
+//  Created by Fouche, George (GE Healthcare) on 11/15/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+import Charts
+import Foundation
+import PredixSDK
+
+/// The Bar class
+public class Bar {
+    var yValues: [Double]
+    var label: String
+    var color: [UIColor]?
+    /// - parameter yValues: Double array of y values
+    /// - parameter label: bar label
+    /// - parameter color: provide a color base on the class UIColor
+    public init(_ yValues: [Double], label: String, color: [UIColor]) {
```

Comment:
"color" parameter is an array of colors, should be plural.
Are the arrays of yValues and colors related?